### PR TITLE
Fix modules build with install prefix (ROOT-10452)

### PR DESCRIPTION
**Pull request by Axel-Naumann**

### CODE REVIEW

File: core/dictgen/src/rootcling_impl.cxx [hunk 1]
```diff
@@ -4180,7 +4180,7 @@ int RootClingMain(int argc,
          clingArgsInterpreter.push_back("-fmodule-map-file=" + modulemap);
 
       clingArgsInterpreter.push_back("-fmodule-map-file=" +
-                                     ROOT::FoundationUtils::GetIncludeDir() +
+                                     std::string(gDriverConfig->fTROOT__GetIncludeDir()) +
```
Comment (vgvassilev) [inline core/dictgen/src/rootcling_impl.cxx]: Why did you need to change this?

**Comment by Axel-Naumann** [comment]
projectroot.roottest.root.aclic.misc.roottest_root_aclic_misc_assertmyfun is fixed by 43721d289d

File: rootx/src/rootx.cxx [hunk 2]
```diff
@@ -201,6 +201,9 @@ static void SetLibraryPath()
 #  endif
    putenv(msg);
 #ifdef ROOTPREFIX
+   } else {
+      char *msg = strdup("LD_LIBRARY_PATH=" ROOTLIBDIR);
```
Comment (amadio) [inline rootx/src/rootx.cxx]: Doesn't this overwrite the `LD_LIBRARY_PATH` for the current process? If this is really needed after the other commits, we should probably try to append or prepend to `LD_LIBRARY_PATH` and think about if Mac/Windows also need to do something here.

Comment (Axel-Naumann) [reply rootx/src/rootx.cxx]: That's fixed in https://github.com/root-project/root/pull/5145 - thanks for the comment!
